components/backup/tests/integrations/test_backup.rs

```diff
@@ -179,6 +186,29 @@ impl TestSuite {
         }
         rx
     }
+
+    fn admin_checksum(&self, backup_ts: u64, start: String, end: String) -> (u64, u64, u64) {
+        let mut checksum = 0;
+        let mut total_kvs = 0;
+        let mut total_bytes = 0;
+        let sim = self.cluster.sim.rl();
+        let engine = sim.storages[&self.context.get_peer().get_store_id()].clone();
+        let snapshot = engine.snapshot(&self.context.clone()).unwrap();
+        let snap_store = SnapshotStore::new(snapshot, backup_ts, IsolationLevel::Si, false);
+        let mut scanner = RangesScanner::new(RangesScannerOptions {
+            storage: TiKVStorage::from(snap_store),
+            ranges: vec![Range::Interval(IntervalRange::from((start, end)))],
+            scan_backward_in_range: false,
+            is_key_only: false,
+            is_scanned_range_aware: false,
+        });
+        while let Some((k, v)) = scanner.next().unwrap() {
```

Comment:
How about using `coprocessor::checksum::ChecksumContext`?

Reply:
use `coprocessor::checksum::ChecksumContext` is more straightforward, but it is hard to construct.